models/organization-extension/src/builder.spec.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import type { TOrganizationExtension } from './types';
+import * as OrganizationExtension from './index';
+
+describe('builder', () => {
+  it(
+    ...createBuilderSpec<TOrganizationExtension, TOrganizationExtension>(
+      'default',
+      OrganizationExtension.random(),
+      expect.objectContaining({
+        id: expect.any(String),
+        createdAt: expect.any(String),
+        updatedAt: expect.any(String),
+        installedApplications: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        installedCustomViews: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        organizationId: expect.any(String),
+        registeredApplications: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        registeredCustomViews: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        applicationsMaintainerContactInformation: expect.objectContaining({
+          id: expect.any(String),
+          email: expect.any(String),
+        }),
+        oidcSsoConfig: expect.objectContaining({
+          id: expect.any(String),
+          clientId: expect.any(String),
+        }),
+      })
+    )
+  );
+  it(
+    ...createBuilderSpec<TOrganizationExtension, TOrganizationExtension>(
+      'graphql',
+      OrganizationExtension.random(),
+      expect.objectContaining({
+        __typename: 'OrganizationExtension',
+        id: expect.any(String),
+        createdAt: expect.any(String),
+        updatedAt: expect.any(String),
+        installedApplications: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        installedCustomViews: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        organizationId: expect.any(String),
+        registeredApplications: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        registeredCustomViews: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        applicationsMaintainerContactInformation: expect.objectContaining({
+          id: expect.any(String),
+          email: expect.any(String),
+        }),
+        oidcSsoConfig: expect.objectContaining({
+          id: expect.any(String),
+          clientId: expect.any(String),
+        }),
```

Comment:
Maybe we could assert on the remaining fields here as well

Reply:
Updated here [69a233f](https://github.com/commercetools/test-data/pull/563/commits/69a233ff8f4162486460a134c7438aac475ce435)